src/sensors/Particles.js

```diff
@@ -61,43 +61,61 @@ ROS3D.Particles = function(options) {
     '}'
     ].join('\n');
 
-    this.geom = new THREE.Geometry();
-    for(var i=0;i<this.max_pts;i++){
-        this.geom.vertices.push(new THREE.Vector3( ));
-    }
-
     var customUniforms =
     {
-        texture:   { type: 't', value: THREE.ImageUtils.loadTexture( texture ) },
+		texture:   { type: 't', value: new THREE.TextureLoader().load( texture ) }
+        //texture:   { type: 't', value: THREE.ImageUtils.loadTexture( texture ) },
```

Comment:
if you comment this code out, it might live there for ever
Git keeps the history, so just remove the line. People can refer to git if they want to know why and when it was remove